lib/index.js

```diff
@@ -48,6 +50,35 @@ $(document).ready(() => {
   })
 })
 
+const addNewFoodsToMeal = () => {
+  var checkedFoods = $('.food-item-checkbox:checkbox:checked')
+  checkedFoods.each(index => {
+    var food = setFoodData(checkedFoods[index])
+    var meal = $(event.target).attr('data')
+    renderAddedFoodToMealTable(meal, food)
+  })
+}
+
+const setFoodData = (food) => {
+  var foodNode = $(food).parents('article')[0]
+  return {
+    id      : $(foodNode).attr('data').split('-')[1],
+    name    : $($(foodNode).find('p')[0]).text(),
+    calories: $($(foodNode).find('p')[1]).text()
+  }
+}
+
+const renderAddedFoodToMealTable = (meal, food) => {
```

Comment:
In diaryFoods.js, we have a function does the same functionality. The method also adds additional information to the button element in the form of a data attribute.

The major difference I see is that renderAddedFoodToMealTable takes a string with the meal name as an argument, and renderFoodToMealTable takes a dinner object as its argument.

If we want to use the method we already have, we'll have to do some formatting to make sure that we can handle either only the string or the entire object.
```
const renderFoodToMealTable = (meal, food) => {
  $(`#${meal.name.toLowerCase()}-table-info`).append(
    `<article class="food-item-${food.id}" id="food-item-row" data="food-${food.id}">
      <p class="food-item-name">${food.name}</p>
      <p class="${meal.name.toLowerCase()}-food-item-calories">${food.calories}</p>
      <div class="button-container">
        <button id="food-item-${food.id}" class="food-item-delete-btn" data="${meal.name.toLowerCase()}-meal" aria-label="Delete">-</button>
      </div>
    </article>`
  )
}
```

Reply:
We would also have to change the method call in line 58 to foodsDiary.renderFoodToMealTable(meal, food)

Reply:
I initially was planning on reusing this method for this ticket but became hesitant as I would be passing in different data than what the function expects (a string rather than an object) - which would deem ${meal.name.toLowerCase()} inflexible. I am open to coming together and finding ways to make this more flexible without affecting the rest of the code that utilizes this function.